package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "scripts": {
+    "clean": "rimraf output && rimraf .pulp-cache",
+    "build": "pulp build -- --censor-lib --strict",
+    "test": "pulp test"
```

Comment:
Haha, nope. I copy-pasted from purescript-nullable. There should probably be some tests though, so I can add some when I get the chance.